lib/components/BuildSeamlessApps.jsx

```diff
@@ -30,7 +30,7 @@ export const BuildSeamlessApps = function() {
                       <b>OpenZeppelin SDK</b> makes smart contract development easy. Save hours of development time by compiling, upgrading, deploying, and interacting with smart contracts with our CLI. <br/><br/>
                     </p>
                     }
-              url='https://docs.openzeppelin.com/openzeppelin/'
+              url='https://docs.openzeppelin.com/learn/sending-gasless-transactions'
```

Comment:
This one seemed fine to me!

Reply:
https://gsn.openzeppelin.com/
Portis links to a GSN specific section, which is why I suggested that OpenZeppelin should do the same.